server/models/MemberInvitation.js

```diff
@@ -151,6 +151,35 @@ function defineModel() {
 
   // ---- Static methods ----
 
+  MemberInvitation.sendEmail = async (memberParams, sequelizeParams, collective, skipDefaultAdmin) => {
```

Comment:
Since we're using `this` in this method (which is the right thing to do), then it's not a static method. It should be above `// ---- Static methods ----`. I'm also surprised that it works with `MemberInvitation.sendEmail =`, it should be `MemberInvitation.prototype.sendEmail =`

Reply:
Interestingly it doesn't work as an instance method with `this`, but it *does* work with your original spec:

```diff
diff --git a/server/models/MemberInvitation.js b/server/models/MemberInvitation.js
index b8744e2c7..82219edee 100644
--- a/server/models/MemberInvitation.js
+++ b/server/models/MemberInvitation.js
@@ -149,9 +149,7 @@ function defineModel() {
     return this.destroy();
   };
 
-  // ---- Static methods ----
-
-  MemberInvitation.sendEmail = async (memberParams, sequelizeParams, collective, skipDefaultAdmin) => {
+  MemberInvitation.prototype.sendEmail = async function (memberParams, sequelizeParams, collective, skipDefaultAdmin) {
     // Load users
     const memberUser = await models.User.findOne({
       where: { CollectiveId: memberParams.MemberCollectiveId },
@@ -180,6 +178,8 @@ function defineModel() {
     });
   };
 
+  // ---- Static methods ----
+
   MemberInvitation.invite = async function (collective, memberParams, { transaction, skipDefaultAdmin } = {}) {
     const sequelizeParams = transaction ? { transaction } : undefined;
 
@@ -214,7 +214,7 @@ function defineModel() {
     });
 
     if (existingInvitation) {
-      await this.sendEmail(memberParams, sequelizeParams, collective, skipDefaultAdmin);
+      await existingInvitation.sendEmail(memberParams, sequelizeParams, collective, skipDefaultAdmin);
       return existingInvitation.update(pick(memberParams, ['role', 'description', 'since']), sequelizeParams);
     }
 
@@ -235,7 +235,7 @@ function defineModel() {
       sequelizeParams,
     );
 
-    await this.sendEmail(memberParams, sequelizeParams, collective, skipDefaultAdmin);
+    await invitation.sendEmail(memberParams, sequelizeParams, collective, skipDefaultAdmin);
 
     return invitation;
   };
```

Do you have a preference?

Reply:
I've updated the code to rely on `this` rather than `memberParams`